Maestro/subscriptions.json

```diff
@@ -519,6 +519,31 @@
         "https://github.com/dotnet/core-setup/blob/release/1.1.0/**/*"
       ],
       "action": "core-setup-pipebuild-release-1.1.0"
+    },
+    // Update dependencies in WCF release/2.0.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/standard/release/2.0.0/Latest.txt"
```

Comment:
Don't actually need the CoreClr dependency, removed it from our dependencies.props
Also, had some issues trying to get core-setup to work right, I'd like to work on that separately.
Will update for corefx.